Xamarin.Forms.GoogleMaps/Xamarin.Forms.GoogleMaps.UWP/MapRenderer.cs

```diff
@@ -341,6 +349,10 @@ void UpdateHasZoomEnabled()
         void UpdateHasScrollEnabled()
         {
         }
+
+        void UpdateHasRotationEnabled()
```

Comment:
I just put it in there for completeness (because HasScrollEnabled and HasZoomEnabled was in there) Take it out if you dont think its needed